catalog/nsili/catalog-nsili-endpoint/src/main/java/com/connexta/alliance/nsili/endpoint/requests/SubmitQueryRequestImpl.java

```diff
@@ -28,20 +32,57 @@
 import com.connexta.alliance.nsili.common.UCO.StringDAGListHolder;
 import com.connexta.alliance.nsili.common.UCO.SystemFault;
 import org.omg.CORBA.NO_IMPLEMENT;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Result;
 
 public class SubmitQueryRequestImpl extends SubmitQueryRequestPOA {
 
+    //TODO TROY
+    private static final String ENCODING = "UTF-8";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SubmitQueryRequestImpl.class);
+
+    private int maxNumReturnedHits = -1;
```

Comment:
Made the -1 constant more obvious that it's for returning all results if the requestor hasn't specified a number.